src/custom_nodes/add_one/Dockerfile.ubuntu

```diff
@@ -19,13 +19,15 @@ RUN apt update && apt install -y build-essential curl
 
 ARG NODE_NAME=add_one
 
+ARG OPS=-O2 -U_FORTIFY_SOURCE -fstack-protector -fno-omit-frame-pointer -D_FORTIFY_SOURCE=1 -fno-strict-overflow
+
 COPY ./queue.hpp ./queue.hpp
 COPY ./common /custom_nodes/common
 COPY . /custom_nodes/${NODE_NAME}/
 COPY custom_node_interface.h /
 WORKDIR /custom_nodes/common
-RUN g++ -c -std=c++17 *.cpp -fpic -Wall -Wno-unknown-pragmas -Werror -Wno-error=sign-compare  -fno-strict-overflow -fno-delete-null-pointer-checks -fwrapv -fstack-protector
+RUN g++ -c -std=c++17 *.cpp -fpic ${OPS} -Wall -Wno-unknown-pragmas -Werror -Wno-error=sign-compare -fno-delete-null-pointer-checks -fwrapv
```

Comment:
Why is `OPS` defined, but not all flags added?
```
-fpic -Wall -Wno-unknown-pragmas -Werror -Wno-error=sign-compare -fno-delete-null-pointer-checks -fwrapv
```

Reply:
So you suggest to add those flags to OPS?